src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/AzureCodeGenerator/AzureAttachConsoleAppCodeGenerator.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.ML.AutoML;
+using Microsoft.ML.CodeGenerator.CSharp;
+using Microsoft.ML.CodeGenerator.Templates.Azure.Console;
+using Microsoft.ML.CodeGenerator.Templates.Console;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML.CodeGenerator.CodeGenerator.CSharp
+{
+    internal class AzureAttachConsoleAppCodeGenerator : IProjectGenerator
+    {
+        private readonly Pipeline _pipeline;
+        private readonly CodeGeneratorSettings _settings;
+        private readonly ColumnInferenceResults _columnInferenceResult;
+        private readonly string _nameSpaceValue;
+
+        public IProjectFileGenerator ModelBuilder { get; private set; }
+        public IProjectFileGenerator PredictProject { get; private set; }
+        public IProjectFileGenerator PredictProgram { get; private set; }
+        public string Name { get; set; }
+
+        public AzureAttachConsoleAppCodeGenerator(Pipeline pipeline, ColumnInferenceResults columnInferenceResults, CodeGeneratorSettings options, string namespaceValue)
+        {
+            _pipeline = pipeline;
+            _settings = options;
+            _columnInferenceResult = columnInferenceResults;
```

Comment:
Does this meet the code style requirements of machinelearning repo? 

If it does, then whatever rules they have are fine. 

If not (I'm not asking to prioritize it now). I just like something of the rules we have over in Model Builder repo. Like explicitly requiring this._pipelines instead of just _pipelines.

Reply:
in machine learning repo, It's quite the opposite. Having prefix `this`  is kind of violating the code style.

Reply:
## Resolved